task8/src/main/java/PostServlets/List.java

```diff
@@ -0,0 +1,49 @@
+package PostServlets;
+import PostWork.Post;
+import PostWork.Service;
+
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+class PostList extends HttpServlet {
+
+    private Service posts = Service.getInstance();
```

Comment:
(cosmetic) Consider giving a more explicit name for the class. In a usual project, there are way more classes that can be named "service", not a single "Service")